Add: armhfp support

Cinux90 opened the pull request:
# Add: armhfp support

Adding yum_repository task to the role to support arm architecture.

## Reference

 - Resolves: #12 
 - See also:

Commit:
Add: EPEL for armhfp

Commit:
Add: EPEL for armhfp

Commit:
Fix: Typos in README.md

Commit:
Fix: Typos in README.md

Commit:
Add: when condition for armhfp

Commit:
Add: when condition for armhfp

Commit:
Fix: Travis Failure
Add: confirm variable to when condition

Commit:
Fix: Travis Failure
Add: confirm variable to when condition

Commit:
Fix: spellcheck

Commit:
Fix: spellcheck

donvipre commented:
Please rewrite your commit messages according to the community templates.
This also applies for the pull request.

donvipre commented on README.md:
```diff
@@ -9,6 +9,8 @@ This role is needed to get access for Extra Packages for Enterprise Linux (EPEL)
 
 ## Installation
 
+**NOTE**: The armhfp repository get automatically rebuilded without any QA and tests. If you want to use it set `wtd_repo_epel_armhfp_confirm` to `true`.
```
I would change it to something like: 

The armhfp repository is automatically rebuilt without any QA or tests.
Please set `wtd_repo_epel_armhfp_confirm` to `true`

donvipre commented on tasks/main.yml:
```diff
@@ -1,7 +1,18 @@
+- name: Setting armhpf EPEL repo to "{{ wtd_repo_epel_state }}"
+  yum_repository:
+    name: 'epel_armhfp'
+    description: 'Epel rebuild for armhfp'
```
EPEL for armhfp

donvipre commented on tests/.aspell.en.pws:
```diff
@@ -12,3 +12,8 @@ Repo
 screenshots
 wtd
 yml
+armhfp
```
please rewrite the whole file into alphabetical order
remove "rebuilded" as this word does not exist.
The attributes in the readme file should have multiline code surrounding them, so maybe epel and repo are also not needed here.

>

donvipre committed:
Polish: README.md used modules links

Commit:
Polish: tests/.aspell.en.pws words sorted alphabetically
Polish: README.md note more unstandable
Polish: tasks/Main.yml better Task name

Commit:
Polish: tests/.aspell.en.pws words sorted alphabetically
Polish: README.md note more unstandable
Polish: tasks/Main.yml better Task name

Cinux90 commented:
@donvipre changes done, please review again. Regarding the spellchecking. Looks like multilinie code surrounding does not help.

donvipre committed:
Merge pull request #15 from while-true-do/update_readme

Polish: README.md used modules links

donvipre commented:
First: rebase needed.

Second thing:
Normally you should add a "cleanup task" which is already there in the yum role in case somebody wants to remove the repository.
I would suggest adding a notify "Cleanup yum metadata" and the dependency to the yum role.

But this is currently a bit difficult because of the whole Ansible Galaxy thing which still doesn't work.
So we should create a second pull request for this.

You can merge after rebase and then create at least a issue for the yum notify dependency

donvipre commented:
As galaxy is finally fixed, please add the notify to the Cleanup yum metadata

donvipre commented on README.md:
````diff
@@ -9,6 +9,8 @@ This role is needed to get access for Extra Packages for Enterprise Linux (EPEL)
 
 ## Installation
 
+**NOTE**: The armhfp repository is automatically rebuilt without any QA or tests. Please set ```wtd_repo_epel_armhfp_confirm``` to ```true```.
````
regarding the false positives in the spellcheck: please use single backticks for inline codeblocks.
After merging the below pull request, they should be ignored correctly.
while-true-do/ansible-galaxy-skeleton#44

donvipre replied:
regarding the false positives in the spellcheck: please use single backticks for inline codeblocks.
After merging the below pull request, they should be ignored correctly.
while-true-do/ansible-galaxy-skeleton#44

QA => quality assurance

"Please set variable to true if you want to use this repository."

donvipre commented on tasks/main.yml:
```diff
@@ -1,7 +1,18 @@
 ---
+- name: Setting EPEL for armhfp to "{{ wtd_repo_epel_state }}"
```
Normally you should add a "cleanup task" which is already there in the yum role in case somebody wants to remove the repository.
I would suggest adding a notify "Cleanup yum metadata" and the dependency to the yum role.
As the Ansible Galaxy thing is now finally fixed, you can depend on the yum role correctly.

donvipre commented on tasks/main.yml:
```diff
@@ -1,7 +1,18 @@
+- name: Setting EPEL for armhfp to "{{ wtd_repo_epel_state }}"
+  yum_repository:
+    name: 'epel_armhfp'
+    description: 'Epel rebuild for armhfp'
```
as description: "Extra Packages for Enterprise Linux 7 rebuilt - armhfp"

donvipre commented on tests/.aspell.en.pws:
```diff
@@ -1,13 +1,17 @@
 Github
 init
 passlib
+QA
```
remove

donvipre reviewed (changes requested):
after a rebase, again some minor things that can be improved.

daniel-schier commented:
Shall I help here?

Cinux90 commented:
Two weeks Holiday == 0 Time ^^'
I will have a look here this weekend.

Commit:
Fix: Update branch with master

Commit:
Add: yum dependency

Commit:
Fix: Spelling

Commit:
Fix: test-whitespace issue

Commit:
Add: handler to remove Cleanup yum via handler
